src/main/java/seedu/recipe/logic/commands/AddCommand.java

```diff
@@ -21,20 +17,26 @@ public class AddCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a recipe to the address book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_DURATION + "DURATION "
+            + PREFIX_PORTION + "PORTION "
```

Comment:
should enclose duration and portion in square brackets since they are optional recipe fields